src/content/docs/infrastructure/install-infrastructure-agent/linux-installation/install-infrastructure-monitoring-agent-linux.mdx

```diff
@@ -299,87 +302,96 @@ To install the infrastructure monitoring agent in Linux, follow these instructio
        ```
        printf "deb [arch=amd64] https://download.newrelic.com/infrastructure_agent/linux/apt focal main" | sudo tee -a /etc/apt/sources.list.d/newrelic-infra.list
        ```
+       <Callout variant="tip">
+         Use [arch=arm64] to install on ARM architectures such us AWS Graviton
```

Comment:
Same as above, `Use [arch=arm64] to install on ARM architectures such us AWS Graviton.`